server/index.js

```diff
@@ -82,29 +83,11 @@ app.use(logger.errorLogger);
 // Final Error Handler
 app.use(logger.fallbackErrorHandler);
 
-// Cluster Activation
-// See: https://nodejs.org/docs/latest-v8.x/api/cluster.html
+// initialize http server instance
+const server = http.createServer(app);
```

Comment:
We have to pass a server instance to terminus rather than the express app instance.